src/test/ui/rfc1598-generic-associated-types/issue-47206.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright 2012 The Rust Project Developers. See the COPYRIGHT
```

Comment:
2018

Reply:
Now that https://github.com/rust-lang/rust/pull/53617 was merged I think the copyright notice can just be removed.

Reply:
I confess that I hate those licenses so much I make a point of leaving the Copyright year untouched (it doesn't matter a whit legally, from what I understand).

Reply:
Woah, I knew nothing about #53617!